corehq/apps/export/export.py

```diff
@@ -128,18 +128,26 @@ def get_export_file(export_instances, filters, progress_tracker=None):
     with writer.open(export_instances):
         for export_instance in export_instances:
             # TODO: Don't get the docs multiple times if you don't have to
-            docs = _get_export_documents(export_instance, filters)
+            docs = _get_export_documents_scroll(export_instance, filters)
             _write_export_instance(writer, export_instance, docs, progress_tracker)
 
     return ExportFile(writer.path, writer.format)
 
 
-def _get_export_documents(export_instance, filters):
+def _get_export_documents_scroll(export_instance, filters):
+    # size here limits each scroll request, not the total number of results
+    return _get_export_documents(export_instance, filters, size=100).scroll()
```

Comment:
ah i see it was what we were using before. probably could bump it to at least 1000